docs/index.rst

```diff
@@ -1,16 +1,44 @@
-.. Honeybee documentation master file, created by
-   sphinx-quickstart on Wed Mar 13 20:09:41 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
 
 Welcome to Honeybee's documentation!
-===================================
+=========================================
+
+.. image:: http://www.ladybug.tools/assets/img/honeybee.png
+
+Honeybee is a collection of Python libraries to create, run and visualize the results of
+daylight `RADIANCE <https://radiance-online.org/>`_ and energy analysis
+`EnergyPlus <https://energyplus.net/>`_ / `OpenStudio <https://www.openstudio.net/>`_.
+
+Installation
+============
+
+To install the core library try ``pip install -U honeybee-core``.
+
+If you want to also include the command line interface try ``pip install -U honeybee-core[cli]``.
+
+To check if the Honeybee command line is installed correctly try ``honeybee viz`` and you
```

Comment:
Does this mean that running `pip install` automatically adds `honeybee` as a terminal/shell command? (if so, super neat!)

Reply:
yes. it is pretty handy. `pip` creates an script for each  `entry_points` in `console_scripts` in `setup.py`. `honeybee` is mapped to `honeybee.cli.main`.

```python
    entry_points={
        "console_scripts": ["honeybee = honeybee.cli:main"]
    }
```